web/styles/compose.css

```diff
@@ -1173,8 +1173,8 @@ textarea.new_message_textarea {
 
     .zulip-icon-send {
         display: block;
-        font-size: 17px;
-        line-height: 16px;
+        font-size: 1.2143em; /* 17px at 14px/em */
+        line-height: 1.1429; /* 16px at 14px/em */
```

Comment:
This isn't the correct math--the switch in `font-size` means the line-height needs to be calculated against 17px.

Reply:
Oops thanks!

Reply:
Fixed and updated screenshots :)